monitoring/deployment_manager/actions/dss/v1/info.py

```diff
@@ -0,0 +1,45 @@
+import base64
+
+from kubernetes.client import V1Secret
+
+from monitoring.deployment_manager.infrastructure import deployment_action, Context
+
+import cryptography.exceptions
+import cryptography.hazmat.backends
+import cryptography.hazmat.primitives.hashes
+import cryptography.hazmat.primitives.serialization
+import cryptography.x509
+import pem
+
+
+def _public_key_bytes(public_key) -> bytes:
+    return public_key.public_bytes(
+        cryptography.hazmat.primitives.serialization.Encoding.PEM,
+        cryptography.hazmat.primitives.serialization.PublicFormat.SubjectPublicKeyInfo)
+
+
+@deployment_action('dss/info/print_ca_public_certs')
+def print_ca_public_certs(context: Context):
```

Comment:
This action should fail gracefully if the namespace doesn't exist. Currently the execution `python3 deployment_manager.py dss/info/print_ca_public_certs examples/test.json` fails with the following trace:
```
Traceback (most recent call last):
  File "[REDACTED]/dss/monitoring/deployment_manager/deployment_manager.py", line 59, in <module>
    sys.exit(main())
  File "[REDACTED]/dss/monitoring/deployment_manager/deployment_manager.py", line 53, in main
    action_method(context)
  File "[REDACTED]/dss/monitoring/deployment_manager/actions/dss/v1/info.py", line 26, in print_ca_public_certs
    ca_crt: V1Secret = context.clients.core.read_namespaced_secret(name='cockroachdb.ca.crt', namespace=context.spec.dss.v1.namespace)
  File "[REDACTED]/dss/monitoring/monitorlib/typing.py", line 156, in __getattribute__
    return self[item]
KeyError: 'dss'
```

Reply:
Done